noir-projects/aztec-nr/aztec/src/macros/functions/call_interface_stubs.nr

```diff
@@ -19,24 +23,27 @@ comptime global FROM_FIELD: TypedExpr = {
 
 comptime global SERIALIZED_ARGS_NAME: Quoted = quote { serialized_args };
 
-pub comptime fn stub_fn(f: FunctionDefinition) -> Quoted {
```

Comment:
The diff became a bit messy here as I dropped the `stub_fn` and put the register_* functions in its place.